and-cli-install.js

```diff
@@ -85,7 +62,105 @@ require("./command-runner").run(async () => {
         },
     };
 
-    // #endregion Functions
+    // #endregion Public Functions
+
+    // -----------------------------------------------------------------------------------------
+    // #region Private Functions
+    // -----------------------------------------------------------------------------------------
+
+    const _bashFileContains = (text) => {
+        return StringUtils.hasValue(
+            shell.cat(file.bashFile()).grep(text).stdout
+        );
+    };
+
+    const _echoInstallErrorAndExit = (code) => {
+        echo.error(`There was an error installing the package: ${code}`);
+        shell.exit(code);
+    };
+
+    const _installAndCliGlobally = () => {
+        const cmd = install.cmd(CLI_NAME);
+        echo.message(
+            `Installing ${CLI_NAME} as global npm tool... (via ${cmd})`
+        );
+
+        const { code } = shell.exec(cmd);
+        if (code !== 0) {
+            _echoInstallErrorAndExit(code);
+            return;
+        }
+
+        echo.success(`Successfully installed ${CLI_NAME} globally`);
+        echo.newLine();
+    };
+
+    const _installDevAlias = () => {
+        echo.message(
+            `Configuring cli development '${CLI_NAME}-dev' bash alias...`
+        );
+        const pathToCli = upath.join(shell.pwd().toString(), ENTRYPOINT);
+        const developerAlias = `alias ${CLI_NAME}-dev='${pathToCli}'`;
+
+        if (_bashFileContains(developerAlias)) {
+            echo.success(`${CLI_NAME}-dev bash alias already installed`);
+            echo.newLine();
+            return;
+        }
+
+        _writeToBashFile("");
+        _writeToBashFile(
+            `# ${CLI_NAME} global development alias pointing to your fork of the repository`
+        );
+        _writeToBashFile(developerAlias);
+
+        echo.success(`Successfully installed ${CLI_NAME}-dev alias`);
+    };
+
+    const _installLocalProjectGlobally = (binName) => {
+        const cmd = install.cmd(".");
+        echo.message(
+            `Installing current project as a global npm tool... (via ${cmd})`
+        );
+
+        const { code } = shell.exec(cmd);
```

Comment:
The code here and on line 88 are identical. This exec and error evaluation could potentially get combined into a shared function.